src/Peachpie.AspNetCore.Web/SynchronizedTextWriter.cs

```diff
@@ -16,6 +16,11 @@ namespace Peachpie.AspNetCore.Web
     /// </summary>
     sealed class SynchronizedTextWriter : TextWriter
     {
+
+#if NETSTANDARD2_0
+        private readonly char[] writeChars;
```

Comment:
maybe just
```c#
readonly char[] writeChars = new char[1];
```
so you don't have to alter the ctor at line 35

Reply:
I tried that, but it's not possible to initialize a char array that way...

Reply:
Ah sorry. I don't know why but when I tried that, I got an error. I must be going crazy... 😓